<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaFunBtnToolBarTKY</class>
 <widget class="QWidget" name="StaFunBtnToolBarTKY">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1781</width>
    <height>42</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>42</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StaFunBtnToolBarTKY</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background-color: rgb(250, 250, 250);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="FunBtn1_RouteBuild">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>进路建立</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn2_TotalCancel">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>总取消</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn3_SignalReopen">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>信号重开</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn4_GuideBtn">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: red;
}
QPushButton:disabled {
	color: gray;	
}</string>
     </property>
     <property name="text">
      <string>引导按钮</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn5_GuideClock">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: red;
}
QPushButton:disabled {
	color: gray;	
}</string>
     </property>
     <property name="text">
      <string>引导总锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn6_TotalRelieve">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: red;
}
QPushButton:disabled {
	color: gray;	
}</string>
     </property>
     <property name="text">
      <string>总人解</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn7_RegionRelieve">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: red;
}
QPushButton:disabled {
	color: gray;	
}</string>
     </property>
     <property name="text">
      <string>区故解</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn8_TotalPosition">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>总定位</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn9_TotalReverse">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>总反位</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn10_SingleLock">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>单锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn11_SingleUnlock">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>单解</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn12_Blockade">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>封锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn13_UnBlockade">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>解封</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn14_FunBtn">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: red;
}
QPushButton:disabled {
	color: gray;	
}</string>
     </property>
     <property name="text">
      <string>功能按钮</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn15_RampUnlock">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>坡道解锁</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn16_PoorRoute">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>分路不良</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn17_IdleConfirm">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>确认空闲</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn18_Lighting">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>点灯</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn19_UnLighting">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>灭灯</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn20_AuxiliaryMenu">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>辅助菜单</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn21_CommandClear">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>命令清除</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn22_CommandIssued">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>命令下达</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="FunBtn23_MethodConvert">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <property name="text">
      <string>方式转换</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
